src/base/rss/rssfeed.cpp

```diff
@@ -399,6 +399,22 @@ void Feed::downloadArticleTorrentIfMatching(const ArticlePtr &article)
     BitTorrent::Session::instance()->addTorrent(torrentUrl, params);
 }
 
+QString Feed::generateSavePath(const QString &savePathTpl, const ArticlePtr &article)
+{
+    QRegExp rx("<(.+)>");
```

Comment:
I'm not sure I'm following, but if you have a more accurate and not too greedy expression, I'd be happy to replace it.